scripts/jest/jest-reporter.js

```diff
@@ -5,8 +5,23 @@ const DefaultReporter = require('jest-cli/build/reporters/default_reporter').def
  * when there are no errors.
  */
 class JestReporter extends DefaultReporter {
-  log(message) {
-    process.stdout.write(message + '\n');
+  constructor(...args) {
+    super(...args);
+
+    this._isLoggingError = false;
+    this.log = message => {
```

Comment:
curious why this syntax in ctor over having this:

```ts
log = (message) => {
   ...
}
```

Reply:
I had to do this to get rid of the "Unexpected token =" error. I don't write plain JS enough to know why that was happening...